### DISCUSSION TITLE
Physical disk error

### DISCUSSION THREAD

**Discussion by licsuj27** [galexrt/dellhw_exporter#56] [Q&A] [answered]
I have 3 physical disks with value showing as 2
dell_hw_storage_pdisk_status 2

 which according to documentation 2 shows that 
2 - Non-Critical, the component is not okay, but not critical.

I confirmed that only 1 disk is actually faulty, therefore I cannot understand how the other 2 are showing as warning.
How is this value gathered?

The last errors I have in /opt/dell/srvadmin/sbin/omreport system esmlog
is from past year. 
Could it be that value is read from the whole log?

**Comment by galexrt** [comment, marked as answer]
The following docs explains how the states are translated from `OK`, `Critical`, etc., into numbers https://github.com/galexrt/dellhw_exporter/blob/main/docs/metrics.md#what-do-the-metrics-mean

***

What does `omreport storage pdisk controller=ID` (replace ID with the controller ID) report?

**Reply by licsuj27** [reply]
ID                              : 0:1:0
Status                          : Non-Critical
ID                              : 0:1:1
Status                          : Non-Critical
ID                              : 0:1:2
Status                          : Non-Critical

**Reply by galexrt** [reply]
As `omreport` is reporting these values the exporter can't really do "anything" about the metrics being set to `2` (`Non-Critical`).

Not sure if there is like a `omreport` reset status command.
